packages/backend/src/api/controllers/tvl/TvlController.ts

```diff
@@ -32,6 +37,16 @@ type TvlResult =
       error: 'DATA_NOT_FULLY_SYNCED' | 'NO_DATA'
     }
 
+type TvlProjectResult =
```

Comment:
This probably should be extracted to other file
`type SuccessResult<T> = { result: 'success', data: T}`
`type ErrorResult<E> = { result: 'error', data: E}`
`type Result<T, E> = SuccessResult<T> | ErrorResult<E>`

And use it in every place where we use this pattern